webapp/src/main/java/com/box/l10n/mojito/metrics/StatsdMetricsConfig.java

```diff
@@ -0,0 +1,59 @@
+package com.box.l10n.mojito.metrics;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/**
+ * StatsD metrics related config
+ */
+@Component
+@ConfigurationProperties("l10n.management.metrics.export.statsd")
+@ConditionalOnProperty(value="management.metrics.export.statsd.enabled", havingValue="true")
```

Comment:
can you split to have a class that only takes care of reading properties: @ConfigurationProperties